os/include/tinyara/pm/pm.h

```diff
@@ -251,8 +251,22 @@
  
 /* This enumeration provides all power management domains */
 enum pm_domain_e {
-	PM_IDLE_DOMAIN = 0,
-	PM_NDOMAINS = CONFIG_PM_NDOMAINS,
+	PM_IDLE_DOMAIN 	= 0,
+#ifdef CONFIG_LCD
+	PM_LCD_DOMAIN 	= 1,
+#endif
+	PM_UART_DOMAIN 	= 2,
+#ifdef CONFIG_WIFI_MANAGER
+	PM_WIFI_DOMAIN 	= 3,
+#endif
+#ifdef CONFIG_BLE_MANAGER
+	PM_BLE_DOMAIN 	= 4,
+#endif
```

Comment:
May I know the reason for the count gap?

Reply:
Hello @edwakuwaku 
we will use fixed domain count 32.

This is for scalability.

Reply:
The PM merely provides the domain to other functions.

the memory and operating time are the same, there is no need to provide the domain by checking whether the config is enabled or disabled

This is the dependence on the user area.